package.json

```diff
@@ -13,7 +13,7 @@
     "dev": "next dev",
     "dev:now": "now dev",
     "build": "next build",
-    "start": "NODE_ENV=production next start",
+    "start": "NODE_ENV=production next start -p $PORT",
     "prod": "npm run build && npm run start",
```

Comment:
The 'start' script is for running the production build locally. Please provide a default PORT here so that the script does not fail:


```
PORT=3000 npm run build && npm run start
```

Reply:
@hakonkrogh done.

Reply:
We've created the `start-platformsh`

Reply:
Great, that resolves it!